src/testUtils/index.tsx

```diff
@@ -15,31 +47,79 @@ export interface QueryString {
     [key: string]: string;
 }
 
-// To-do: improve this API.
-// shallowRender should default to true,
-// shouldn't have to pass an empty propOverrides object to get a shallow-rendered wrapper with no prop overrides, e.g.:
-// const { wrapper } = setup({}, true);
+export const contextTypes: { [key: string]: Function } = {
+    store: object,
+    endpoints: endpointsPT,
+    languages: languagesPT,
+    flow: flowPT
+};
+
+export const baseState: AppState = {
+    ...initialState,
+    ...{
+        flowContext: {
+            ...initialState.flowContext,
+            definition: require('../../__test__/flows/colors.json') as FlowDefinition
+        }
+    },
+    ...{
+        flowEditor: {
+            ...initialState.flowEditor,
+            editorUI: {
+                ...initialState.flowEditor.editorUI,
+                language: getBaseLanguage((config as FlowEditorConfig).languages)
+            }
+        }
+    }
+};
+
+export const configProviderContext: ConfigProviderContext = {
```

Comment:
exporting an object here that contains only the configuration properties the app cares about